metaflow/client/filecache.py

```diff
@@ -30,18 +31,13 @@ def __init__(self, metadata=None, event_logger=None, monitor=None,
         self._monitor = monitor
 
         self._objects = None
-        # We will have a separate blob_cache per flow and datastore type
-        # This is to respect the fact that the keys across datastore types
-        # and/or flows may not be unique (across flows is less likely but we are
-        # keeping in line with the fact that datsatores are flow based)
-        self._blob_caches = {}
```

Comment:
Why did you remove this? This can cause errors potentially (not right now but there is no guarantee that keys are unique across datastores and there is the possibility that multiple datastores are accessed by this file cache).

Reply:
Reverting